openmetadata-ui/src/main/resources/ui/src/components/SchemaTab/SchemaTab.interfaces.ts

```diff
@@ -28,6 +29,7 @@ export type Props = {
   hasTagEditAccess: boolean;
   isReadOnly?: boolean;
   entityFqn: string;
+  tablePartitioned?: TablePartition;
```

Comment:
```suggestion
  tablePartitioned: Table['tablePartition']
```

Reply:
Since it's a custom interface will stick to direct type assignment instead of infer